src/SprykerEco/Yves/Payone/Plugin/PayoneKlarnaSubFormPlugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Payone\Plugin;
+
+use Spryker\Yves\Kernel\AbstractPlugin;
+use Spryker\Yves\StepEngine\Dependency\Form\StepEngineFormDataProviderInterface;
+use Spryker\Yves\StepEngine\Dependency\Plugin\Form\SubFormPluginInterface;
+use SprykerEco\Yves\Payone\Form\AbstractPayoneSubForm;
+
+/**
+ * @method \SprykerEco\Yves\Payone\PayoneFactory getFactory()
+ */
+class PayoneKlarnaSubFormPlugin extends AbstractPlugin implements SubFormPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Creates KlarnaSubForm subform.
+     *
+     * @api
+     *
+     * @return \SprykerEco\Yves\Payone\Form\AbstractPayoneSubForm
+     */
+    public function createSubForm(): AbstractPayoneSubForm
+    {
+        return $this->getFactory()->createKlarnaSubForm();
+    }
+
+    /**
+     * {@inheritDoc}
+     * - Creates KlarnaSubFormDataProvider data provider.
```

Comment:
* Add a backtick to the name of the data provider.

Reply:
++